views/index.html

```diff
@@ -6,15 +6,20 @@
 <head>
 	<meta charset="utf-8" />
 	<title>BirdNET-Go Dashboard</title>
-	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
-	<link rel="icon" href="data:,"> <!-- Blank favicon to prevent 404 errors -->
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, viewport-fit=cover">
+	<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/favicon-32x32.png">
```

Comment:
On iPhone, `viewport-fit=cover` will make use of the full viewport on landscape. This way there's no need to alter the Tailwind breakpoints only for those devices.